packages/wonder-blocks-dropdown/components/option-item.js

```diff
@@ -184,6 +196,9 @@ const styles = StyleSheet.create({
         whiteSpace: "nowrap",
         userSelect: "none",
         marginLeft: Spacing.xSmall,
+        // added to truncate strings that are longer than expected
+        overflow: "hidden",
+        textOverflow: "ellipsis",
```

Comment:
I recall we chatted about using `title`. Did you get a chance to test with screen reader? Is there any way to add them but not for screen readers so that we don't have redundant labels?

Reply:
Not yet :(. Also, I noticed that we don't support `aria-label` and or `title` in option items, so I'm going to work on that functionality next sprint as part of this ticket: https://khanacademy.atlassian.net/browse/WB-615